clang/include/clang/Driver/Options.td

```diff
@@ -2514,6 +2514,18 @@ def fsanitize_undefined_strip_path_components_EQ : Joined<["-"], "fsanitize-unde
            "when emitting check metadata.">,
   MarshallingInfoInt<CodeGenOpts<"EmitCheckPathComponentsToStrip">, "0", "int">;
 
+  def sanitize_targets_EQ
+      : Joined<["-"], "fsanitize-target=">,
+        MetaVarName<"<target>">,
+        Visibility<[ClangOption, CC1Option]>,
```

Comment:
Should this be available for Windows as well?  If so, need to add `CLOption` to the list of `Visibility` values.

Reply:
Hi, @mdtoguchi 
We only support device sanitizer for Linux target now.
Thanks very much.